README.md

```diff
@@ -120,12 +120,33 @@ npm run dev             # start a local dev server for Storybook
 npm run lint            # run linter
 ```
 
-### Maintain Library
+## Publishing new versions
 
-```bash
-npm run compile           # generate lib folder for publishing to npm
-npm run build-storybook   # generate storybook static bundle for deploy with GH Pages
-```
+Dispatch the [Publish](https://github.com/Constructor-io/constructorio-ui-quizzes/actions/workflows/publish.yml) workflow in GitHub Actions. You're required to provide two arguments:
+
+- **Version Strategy**: `major`, `minor`, or `patch`.
+- **Title**: A title for the release.
+
+This workflow will automatically:
+
+1. Bump the library version using the provided strategy.
+2. Create a new git tag.
+3. Create a new GitHub release.
+4. Compile the library.
+5. Publish the new version to NPM.
+6. (_disabled for now_) Publish the new version to our public CDN.
+7. Deploy the Storybook docs to GitHub Pages.
+8. Report the progress on the [relevant Slack channel](https://constructor.slack.com/archives/C061D3CFVR9).
```

Comment:
**Suggestion:** Let's remove this

```suggestion
6. Deploy the Storybook docs to GitHub Pages.
7. Report the progress on the [relevant Slack channel](https://constructor.slack.com/archives/C061D3CFVR9).
```

Reply:
Good catch! 16ddb833c50cf5393bcbee466d57f79a47641603